test/Microsoft.Framework.PackageManager.FunctionalTests/DnuListTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Framework.FunctionalTestUtils;
+using Xunit;
+
+namespace Microsoft.Framework.PackageManager.FunctionalTests
+{
+    public class DnuListTests : IClassFixture<DnuListTestContext>, IDisposable
+    {
+        private readonly DnuListTestContext _context;
+        private readonly DisposableDir _workingDir;
+
+        public DnuListTests(DnuListTestContext context)
+        {
+            _context = context;
+        }
+
+        public static IEnumerable<object[]> RuntimeComponents
+        {
+            get { return TestUtils.GetRuntimeComponentsCombinations(); }
+        }
+
+        public void Dispose()
+        {
+            _workingDir.Dispose();
+        }
+
+        [Theory]
+        [MemberData("RuntimeComponents")]
+        public void DnuList_EmptyProject_Default(string flavor, string os, string architecture)
+        {
+            var runtimeHomePath = _context.GetRuntimeHome(flavor, os, architecture);
+
+            var projectJson = Path.Combine(_workingDir.DirPath, "project.json");
+            File.WriteAllText(projectJson, @"{}");
+
+            string stdOut, stdErr;
+            Assert.Equal(0, DnuTestUtils.ExecDnu(runtimeHomePath, "list", "", out stdOut, out stdErr, environment: null, workingDir: _workingDir.DirPath));
```

Comment:
Dnu list's output is too big to build a stable expected string. The minimal looks like this:

```
Listing dependencies for dnx-issue1644 (C:\myk\issue_repos\dnx-issue1644\project.json)

[Target framework DNX,Version=v4.5.1 (dnx451)]

  framework/Microsoft.CSharp 4.0.0.0
    -> dnx-issue1644 1.0.0
  framework/mscorlib 4.0.0.0
    -> dnx-issue1644 1.0.0
  framework/System 4.0.0.0
    -> dnx-issue1644 1.0.0
  framework/System.Core 4.0.0.0
    -> dnx-issue1644 1.0.0
```

The dependencies may change without any change to dnu list. To my experience it's not a good idea to build a test too rigid that it fails without any changes to the code it tests.

Plus there are tests do check the output. Checkout other test cases in the same class.